src/content/learn/creating-a-react-app.md

```diff
@@ -55,61 +76,119 @@ If you're new to Expo, check out the [Expo tutorial](https://docs.expo.dev/tutor
 Expo is maintained by [Expo (the company)](https://expo.dev/about). Building apps with Expo is free, and you can submit them to the Google and Apple app stores without restrictions. Expo additionally provides opt-in paid cloud services.
 
 
-## Other options {/*other-options*/}
+### Other Frameworks {/*other-frameworks*/}
```

Comment:
what makes these frameworks "other"?

the following sentence suggests its because they are up-and-coming (i.e. early stage right now). is that true?

it may be worth briefly noting why these don't make the "main list" (not necessarily individually, just what makes this overall section its own section)